app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -12,22 +12,20 @@
 
 import org.mozilla.focus.session.Session;
 import org.mozilla.gecko.GeckoView;
-import org.mozilla.gecko.GeckoViewSettings;
+import org.mozilla.gecko.GeckoSession;
+import org.mozilla.gecko.GeckoSession.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- org\.mozilla\.gecko\.GeckoSession\.\*\.